tests/Unit/SymfonyCache/CustomTtlListenerTest.php

```diff
@@ -87,6 +87,23 @@ public function testNoCustomTtl()
         $this->assertFalse($response->headers->has(CustomTtlListener::SMAXAGE_BACKUP));
     }
 
+    public function testNoCustomTtlNoFallback()
+    {
+        $ttlListener = new CustomTtlListener('X-Reverse-Proxy-TTL', false, true);
```

Comment:
```suggestion
        $ttlListener = new CustomTtlListener('X-Reverse-Proxy-TTL', false, false);
```
(adjust to my inversion of the flag)